pkg/multicluster/virtual_cluster.go

```diff
@@ -62,12 +63,12 @@ func InitClusterInfo(cfg *rest.Config) error {
 		if err != nil {
 			return errors.Wrap(err, "fail to get registered clusters")
 		}
-		for _, cluster := range clusters.Items {
+
+		velaslices.ParFor(clusters.Items, func(cluster v1alpha1.VirtualCluster) {
 			if err = SetClusterVersionInfo(ctx, cfg, cluster.Name); err != nil {
 				klog.Warningf("set cluster version for %s: %v, skip it...", cluster.Name, err)
-				continue
 			}
-		}
+		}, velaslices.Parallelism(len(clusters.Items)))
```

Comment:
clusters.Item could be too large for parallelism. Let's remove it and use the default parallelism.

Reply:
done